src/patterns.rs

```diff
@@ -56,8 +56,15 @@ impl Rewrite for Pat {
                     None
                 }
             }
-            PatKind::Range(ref lhs, ref rhs) => {
-                rewrite_pair(&**lhs, &**rhs, "", "...", "", context, shape)
+            PatKind::Range(ref lhs, ref rhs, ref end_kind) => {
+                match *end_kind {
+                    RangeEnd::Included => {
+                        rewrite_pair(&**lhs, &**rhs, "", "...", "", context, shape)
+                    }
+                    RangeEnd::Excluded => {
+                        rewrite_pair(&**lhs, &**rhs, "", "..", "", context, shape)
```

Comment:
Honestly, I'm not sure about this one... I don't understand code that much to understand this particular part.